packages/frontend-2/components/feedback/Dialog.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <LayoutDialog
+    v-model:open="isOpen"
+    title="Give us feedback"
+    :buttons="dialogButtons"
+    max-width="md"
+  >
+    <div class="flex flex-col gap-4">
+      <p class="text-body-xs text-foreground font-medium">
+        How can we improve Speckle? If you have a feature request, please also share how
+        you would use it and why it's important to you
+      </p>
+      <FormTextArea
+        v-model="feedback"
+        :rules="[isRequired]"
+        name="feedback"
+        label="Feedback"
+        color="foundation"
+      />
+    </div>
+  </LayoutDialog>
+</template>
+
+<script setup lang="ts">
+import type { LayoutDialogButton } from '@speckle/ui-components'
+import { useForm } from 'vee-validate'
+import { useMixpanel } from '~/lib/core/composables/mp'
+import { useZapier } from '~/lib/core/composables/zapier'
+import { useGlobalToast, ToastNotificationType } from '~~/lib/common/composables/toast'
+import { isRequired } from '~/lib/common/helpers/validation'
+import { useActiveUser } from '~~/lib/auth/composables/activeUser'
+
+type FormValues = { feedback: string }
+
+const isOpen = defineModel<boolean>('open', { required: true })
+
+const { activeUser: user } = useActiveUser()
+const mixpanel = useMixpanel()
+const { sendWebhook } = useZapier()
+const { triggerNotification } = useGlobalToast()
+const { handleSubmit } = useForm<FormValues>()
+
+const feedback = ref('')
+
+const dialogButtons = computed((): LayoutDialogButton[] => [
+  {
+    text: 'Send',
+    props: { color: 'primary' },
+    onClick: () => {
+      onSubmit()
```

Comment:
Doesn't make sense that there's `useForm` and yet you're not actually using a `<form/>`. Use the on-submit prop of LayoutDialog to properly use form submission for this and convert the button to a submit button.

Reply:
Good point, updated 👍